### PR TITLE
Add pager buttons on manual pages

### LINKED ISSUE #2: Add Previous and Next buttons on Manual page
Would be nice if the Manual pages can be navigated through a *Previous page* and a *Next page* buttons. We can use the `pager` flag used on [urubu-quickstart](https://github.com/jandecaluwe/urubu-quickstart/):

```html
    {% if this.pager %}
    <ul class="pager">
      {% if this.prev %}
      <li class="previous"><a href="{{this.prev.url}}">&larr; {{this.prev.title}}</a></li>
      {% endif %}
      {% if this.next %}
      <li class="next"><a href="{{this.next.url}}">{{this.next.title}} &rarr;</a></li>
      {% endif %}
    </ul>
    {% endif %}
```
Original code: https://github.com/jandecaluwe/urubu-quickstart/blob/ab695aa5be7dbee38da758639150fba70389c13b/_layouts/page.html#L32-L41

The idea came up after #1 .

### PR DESCRIPTION
**Pull request by santisoler**
Add buttons for navigating to the previous and next manual page after
its body text.

Fixes #2

### CODE REVIEW

**Comment by santisoler** [comment]
@leouieda See if you like how I set up the pager buttons. I put them above the reference to the Lab Carpentry blueprints. I'm having doubts whether to add the buttons to the Manual `index.md`: because it doesn't have `previous` or `next` pages, adding `pager: true` is useless, therefore I would have to modify the layout.

File: _layouts/manual.html [hunk 1]
```diff
@@ -17,6 +17,18 @@
                 <h1>{{ this.title }}</h1>
             {% endif %}
             {{ this.body }}
+            {% if this.pager is defined %}
```
Comment (leouieda) [inline _layouts/manual.html]:
```suggestion
            {% if this.pager is defined and this.pager %}
```

**Comment by santisoler** [comment]
> @santisoler the layout looks good to me! If you want, you can collapse those two `if` statements into 1 but if not I'm fine with merging as is.

Done @leouieda! I've also added the pager to the main Manual page pointing towards the first page on its content. I think this is good to me, if you agree please feel free to merge it.

**Comment by leouieda** [comment]
:+1: looks great to me!

**Pull request merged**